src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -638,6 +639,11 @@ public class SimpleEvents {
 					"\tcancel the event")
 				.since("INSERT VERSION");
 		}
+		Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "unknown command")
```

Comment:
This is a Paper exclusive event. Needs a Skript.classExists method check before registering.